substrate/utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -690,10 +695,19 @@ impl PalletCmd {
 		&self,
 		state: &'a BenchmarkingState<H>,
 	) -> Result<FetchedCode<'a, BenchmarkingState<H>, H>> {
-		log::info!("Loading WASM from state");
-		let state = sp_state_machine::backend::BackendRuntimeCode::new(state);
-
-		Ok(FetchedCode { state })
+		if let Some(runtime) = self.runtime.as_ref() {
+			log::info!("Loading WASM from file");
```

Comment:
should there be a custom log target for this?

Reply:
Yea will change for consistency.  
Not sure if they make the most sense though, since the default is the crate, which is probably good enough for most cases.